doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -468,11 +631,24 @@ Response code:
 
 + Response 204
 
+### Update existing entity attributes [PATCH /v2/entities/type/{entityType}/id/{entityId}]
+
+The request payload is an object representing the attributes to update. The object follows
+the JSON entity representation format (described in a section above), except that `id` and `type`
+are not allowed.
```

Comment:
well, that's part of a separated discussion. I'm still in doubt we need a separate resource for  

/type/myType/id/myId

 as we can do the same with /entities/myId?type=xxx 

and that would save us from complexity in the spec